scripts/make_huntress.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Make a TreeNode tree given a mutation matrix
+to generate a huntress tree.
+
+Example Usage:
+poetry run python ../scripts/make_huntress.py
+
+Example Usage with arguments:
+    poetry run python scripts/plot_trees.py
+    --data_fp data/mutation_matrices/mark00
+    --out_dir data/plots/tree/mark00
+
+"""
+
+import argparse
+import json
+
+from pathlib import Path
+
+from pyggdrasil.tree import TreeNode
+import pyggdrasil.serialize as serialize
+
+
+from pyggdrasil.tree_inference import (
+    huntress_tree_inference,
+    CellSimulationId,
+    MutationDataId,
+    TreeId,
+    TreeType,
+)
+
+
+#############################################
+# Placeholder Fns implemented in PR #42
+
+
+class CellSimulationData:
+    """TypedDict representing the data output of cell_simulation.py"""
+
+    raise NotImplementedError
+
+
+def get_simulation_data(data: dict):
+    """Load the mutation matrix from json object of
+    the simulation data output of cell_simulation.py
+    Args:
+        data: dict
+            data dictionary containing - serialised data
+    Returns:
+        TypedDict of: CellSimulationData
+            adjacency_matrix: interface.TreeAdjacencyMatrix
+                Adjacency matrix of the tree.
+            perfect_mutation_mat: PerfectMutationMatrix
+                Perfect mutation matrix.
+            noisy_mutation_mat: interface.MutationMatrix
+                Noisy mutation matrix. May be none if cell simulation was errorless.
+            root: TreeNode
+                Root of the tree.
+    """
+    raise NotImplementedError
+
+
+#############################################
+
+
+def create_parser() -> argparse.Namespace:
+    """
+    Parser for required input user.
+
+    Returns:
+        args: argparse.Namespace
+    """
+
+    parser = argparse.ArgumentParser(
+        description="Generate a huntress tree from a mutation matrix."
+    )
+
+    parser.add_argument(
+        "--out_dir",
+        required=True,
+        help="Output directory to save the TreeNode trees.",
+        type=str,
+    )
+
+    parser.add_argument(
+        "--data_fp",
+        required=True,
+        help="data_fp to load the mutation matrix",
+        type=str,
+    )
+
+    parser.add_argument(
+        "--n_nodes",
+        required=True,
+        help="number of nodes in the tree, i.e. no of nodes - 1 = mutations",
+    )
+
+    parser.add_argument(
+        "--fpr",
+        required=True,
+        help="False positive rate of data",
+    )
+
+    parser.add_argument(
+        "--fnr",
+        required=True,
+        help="False negative rate of data",
+    )
+
+    args = parser.parse_args()
+
+    return args
+
+
+def main() -> None:
+    """
+    Main function for plotting trees from MCMC samples.
+
+    Returns:
```

Comment:
In general, I don't think `Returns None` in the docstring is needed, as it can be read from the type hint :wink: